channel/state.go

```diff
@@ -91,6 +84,17 @@ func newState(params *Params, initBals Allocation, initData Data) (*State, error
 	}, nil
 }
 
+// ValidateState validates that the Allocation is valid for the given number of
+// participants, so that they can be used to construct a valid State.
+func ValidateState(numParts int, alloc Allocation) error {
```

Comment:
Why is the function called `ValidateState` when it in fact validates an allocation?
```suggestion
func ValidateAllocation(numParts int, alloc Allocation) error {
```

Reply:
Because it was the only validation taking place in `newState`. But I agree, it could actually be called `ValidateAllocation`.